node-data/src/ledger.rs

```diff
@@ -339,6 +343,23 @@ impl PartialEq<Self> for SpentTransaction {
 
 impl Eq for SpentTransaction {}
 
+/// Defines a set of certificates of any former iterations
+#[derive(Default, Eq, PartialEq, Clone)]
+#[cfg_attr(any(feature = "faker", test), derive(Dummy))]
+pub struct IterationsInfo {
+    /// Represents a list of certificates where position is the iteration
+    /// number
+    pub cert_list: Vec<Option<Certificate>>,
```

Comment:
Would this be more efficient as a map (taking into account there might be multiple missing iterations)?
It's just a thought... I guess missing iterations should not be very common (unless the node is eclipsed)